airframe-surface/.jvm/src/test/scala-2/wvlet/airframe/surface/AirSpecBridgeCompat.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package wvlet.airframe.surface
+
+object AirSpecBridgeCompat {
```

Comment:
Note: AirSpec has been released for all Scala {2, 3} x {JVM, JS} combinations, so we no longer need this bridge to munit.